### ISSUE TITLE
Bug: error when defining fallbacks in `var` in `no-invalid-properties`

### ISSUE THREAD

**Issue by Tanujkanti4441** [eslint/css#180]
### Environment

ESLint version: 
@eslint/css version: latest
Node version: latest
npm version: latest
Operating System: windows

### Which language are you using?

stylesheet

### What did you do?

```css
/* eslint css/no-invalid-properties: ["error"] */

:root {
	--my-color: red;
}

a {
	color: var(--my-color, red);
}

.two {
	color: var(--my-color, var(--my-background, pink)); 
}
```

### What did you expect to happen?

No error as variable `--my-color` is defined already.

### What actually happened?

`Unknown property 'color' found  css/no-invalid-properties`

### Link to Minimal Reproducible Example

none

### Participation

- [x] I am willing to submit a pull request for this issue.

**Comment by nzakas** [comment]
Nice catch.
